sticker/src/tensorflow/tagger.rs

```diff
@@ -25,7 +25,7 @@ use crate::{SentVectorizer, Tag, TopK, TopKLabels};
 pub struct ModelConfig {
     /// Model batch size, should be kept constant between training and
```

Comment:
Why should batch size be constant between train and prediction?

Reply:
I have no idea. Maybe this was read from the TOML configuration and then hardcoded in the graph at some point?